app/src/main/java/org/listenbrainz/android/ui/navigation/AppNavigation.kt

```diff
@@ -44,10 +44,6 @@ fun AppNavigation(
     snackbarState: SnackbarHostState,
 ) {
     fun NavOptionsBuilder.defaultNavOptions() {
-        // Avoid building large backstack
-        popUpTo(AppNavigationItem.Feed.route) {
```

Comment:
We want this option to be here, rather hardcode options into each use case (artist, album and playlist) without this field.